packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -605,23 +606,22 @@ export function getPreviewMiniflareOptions(
 	};
 }
 
-const removedMessages = [/^Ready on http/, /^Updated and ready on http/];
-
 /**
  * A Miniflare logger that forwards messages onto a Vite logger.
  */
 class ViteMiniflareLogger extends Log {
 	private logger: vite.Logger;
-	constructor(config: vite.ResolvedConfig) {
+	constructor(
+		config: vite.ResolvedConfig,
+		private isPreview = false
```

Comment:
Can you add a comment documenting what this means?
I guess `isPreview` means that it is running `vite dev` rather than `vite preview`?

Reply:
Nice idea! I'll try that.